jest.config.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
     escapeString: true,
     printBasicPrototype: true,
   },
+  workerIdleMemoryLimit: '512MB',
```

Comment:
We can try with 1GB, or 2GB as well to see if it improves the time but still makes the memory in check? Since we have 8GB instance.

Reply:
Set it to 1GiB just to check, plus added heap usage logging just for testing